CHANGELOG.md

```diff
@@ -45,6 +45,9 @@
 - `multihash`: Removed the crate in favour of the upstream crate.
   [PR 1472](https://github.com/libp2p/rust-libp2p/pull/1472)
 
+- `libp2p-floodsub`: Allow sent messages seen as subscribed
```

Comment:
Please merge master and move this at the top, since we published a version in between!

Reply:
Done and done.